docs/minimal_app.rst

```diff
@@ -0,0 +1,23 @@
+Minimal example application
+----------------------------
+
+Please have a look at [The examples](https://github.com/jugmac00/flask-reuploaded/tree/master/examples)
+
+..    
+    _Comment: _Comment: This link is broken at time of writing. 
+
+Running the example application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to run the application,
+you have to enter the following commands...
+
+.. code-block:: bash
+    
+    ❯ git clone https://github.com/jugmac00/flask-reuploaded.git
+    
+    ❯ cd flask-reuploaded/examples
+
+    ❯ python3 app1.py # or app2.py
+
+Then point your browser to `http://127.0.0.1:5000/`
```

Comment:
This won't work, as you need to create a virtual environment, install the dependencies into it, and only then run it.